ISISComputingGroup/EPICS-motionSetPoints
# Ticket 6423: motion set points move on start

See https://github.com/ISISComputingGroup/IBEX/issues/6423

Issue was that setting the units on start up was using PP and so causing the motor to process

To test:
* Checkout https://github.com/ISISComputingGroup/EPICS-ioc/pull/615
* Cherry-pick https://github.com/ISISComputingGroup/EPICS-motionSetPoints/pull/22/commits/0c10e2837d330d33e045cdc4324d34718135a930 on to master
* Run https://github.com/ISISComputingGroup/EPICS-IOC_Test_Framework/pull/453 and confirm it fails (e.g. that the motor does move when the ioc is restarted 
* Checkout this branch and rebuild
* Confirm the test now passes
